### PR TITLE
disaable stats from CI

### PR DESCRIPTION
**Pull request by edublancas**
I disabled the stats in the CI. Needed to change some of the telemetry_tests (just minor changes). Please review for your awareness.

I also saw you're working on the stats improvements, once this PR is merged, you may want to rebase to get the new `test_telemetry.py`

### CODE REVIEW

File: tests/telemetry/test_telemetry.py [hunk 1]
```diff
@@ -40,11 +60,9 @@ def test_opt_str_validation():
     assert type(exception_raised) == TypeError
 
 
-# Test class functions
-def test_check_stats_enabled():
+def test_check_stats_enabled(ignore_env_var_and_set_tmp_default_home_dir):
     stats_enabled = telemetry.check_stats_enabled()
-    assert isinstance(stats_enabled, bool)
-    assert stats_enabled
+    assert stats_enabled is True
```
Comment (idomic) [inline tests/telemetry/test_telemetry.py:65]: What's wrong with asserting the inner value?

**Review by idomic — APPROVED** [review]
lgtm besides this inline comment above

File: tests/telemetry/test_telemetry.py (hunk 1 above)
Comment (edublancas) [reply tests/telemetry/test_telemetry.py:65]: nothing wrong but `statis_enabled is True` covers the previous two lines (is bool and is True), it's just shorter and more explicit :)

**Comment by edublancas** [comment]
re: ploomber scaffold. good catch. I haven't applied it yet. will do!

**Pull request merged**